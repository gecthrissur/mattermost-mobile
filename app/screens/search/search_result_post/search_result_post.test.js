// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
// See LICENSE.txt for license information.

import React from 'react';

import {shallowWithIntl} from 'test/intl-test-helper';

import SearchResultPost from './search_result_post';

describe('SearchResultPost', () => {
    const baseProps = {
        goToThread: jest.fn(),
        onHashtagPress: jest.fn(),
        onPermalinkPress: jest.fn(),
        previewPost: jest.fn(),
        postId: 'post-id',
        isDeleted: false,
        highlightPinnedOrFlagged: false,
        managedConfig: {},
        showFullDate: false,
        skipFlaggedHeader: false,
        skipPinnedHeader: false,
    };

    test('should match snapshot', async () => {
        const wrapper = shallowWithIntl(<SearchResultPost {...baseProps}/>);

        expect(wrapper.getElement()).toMatchSnapshot();
    });
});
