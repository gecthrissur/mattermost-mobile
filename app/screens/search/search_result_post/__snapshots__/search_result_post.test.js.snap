// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchResultPost should match snapshot 1`] = `
<Connect(Post)
  highlightPinnedOrFlagged={false}
  isSearchResult={true}
  managedConfig={Object {}}
  onHashtagPress={[MockFunction]}
  onPermalinkPress={[MockFunction]}
  onPress={[MockFunction]}
  onReply={[MockFunction]}
  postId="post-id"
  shouldRenderReplyButton={true}
  showAddReaction={false}
  showFullDate={false}
  skipFlaggedHeader={false}
  skipPinnedHeader={false}
  testID="search_result_post.post"
/>
`;
